src/Service/Security/SecurityChecker/SensioLabsSecurityChecker.php

```diff
@@ -39,6 +40,11 @@ public function update(): void
         $this->updateRepo();
     }
 
+    public function hasDbBeenUpdated(): bool
+    {
+        return $this->dbUpdated;
```

Comment:
please no, service should not have a state, better return object or bool from `update` method